programs/x509/cert_write.c

```diff
@@ -286,6 +316,52 @@ int main( int argc, char *argv[] )
         {
             opt.serial = q;
         }
+        else if( strcmp( p, "authority_identifier" ) == 0 )
+        {
+            opt.authority_identifier = atoi( q );
+            if( opt.authority_identifier != 0 &&
+                opt.authority_identifier != 1 )
+            {
+                goto usage;
```

Comment:
@gilles-peskine-arm I'll add that. 

What's your take on the scope of the backports?

Reply:
@gilles-peskine-arm Done.

Reply:
@hanno-arm This can be backported but doesn't need to. Whichever expends the least energy, i.e. don't explicitly backport it but don't cut it out if it's part of a patch that you're backporting.

Reply:
@gilles-peskine-arm I already prepared minimal backports, #1108 and #1109 - could you have a look and check if you think they should be extended? @andresag01 raised some concerns there.